build.sbt

```diff
@@ -207,6 +207,9 @@ lazy val V = new {
   val coursier = "2.0.0-RC6-21"
   val coursierInterfaces = "0.0.22"
   val ammonite = "2.1.4-6-2179b35"
+  val mill = "0.7.3"
+  val gradle = "6.1.0"
```

Comment:
I didn't set this to the latest since the [Bloop site](https://scalacenter.github.io/bloop/docs/build-tools/gradle) mentions that the latest test/supported in 6.1